src/Essentials/src/AppInfo/AppInfo.android.cs

```diff
@@ -16,8 +16,6 @@ class AppInfoImplementation : IAppInfo
 #pragma warning disable CS0618, CA1416, CA1422 // Deprecated in API 33: https://developer.android.com/reference/android/content/pm/PackageManager#getPackageInfo(java.lang.String,%20int)
 		static readonly Lazy<PackageInfo> _packageInfo = new Lazy<PackageInfo>(() => Application.Context.PackageManager.GetPackageInfo(_packageName.Value, PackageInfoFlags.MetaData));
 #pragma warning restore CS0618, CA1416, CA1422
-		static readonly Lazy<AppTheme> _requestedTheme = new Lazy<AppTheme>(GetRequestedTheme);
-		static readonly Lazy<LayoutDirection> _layoutDirection = new Lazy<LayoutDirection>(GetLayoutDirection);
```

Comment:
We can't cache these as things in the `Application.Context.Resources.Configuration` are all dynamic - changeable by both the use and the OS internal processing.